india_compliance/gst_india/utils/gstin_info.py

```diff
@@ -56,6 +57,26 @@ def get_gstin_info(gstin):
     return gstin_info
 
 
+def update_gstin_details(response):
+    registration_date = response.rgdt
+    cancelled_date = response.cxdt
+
+    try:
+        cancelled_date = parse_datetime(cancelled_date, day_first=True)
+    except Exception:
+        cancelled_date = None
```

Comment:
Better with if condition.

If cancelled_date: ...

Reply:
But sometimes the cancelled_date have value like : "-"